test/functional/fixtures/regression/gh-851/test.js

```diff
@@ -0,0 +1,17 @@
+describe('[Regression](GH-851)', function () {
+    it('Should raise click for common parent element', function () {
+        return runTests('testcafe-fixtures/index.test.js', 'Raise click for common parent', { skip: ['firefox', 'firefox-osx'] });
```

Comment:
it's all ok, but I suppose we should add separate tests for `firefox` since it has a different behavior